ui/src/app/app-routing.module.ts

```diff
@@ -39,8 +39,10 @@ const routes: Routes = [
   {
     path: 'dashboard',
     component: DashboardComponent,
-    //TODO: (zach) dashboard need a query param of project id before it can be activated
+    //TODO: (zach) if the projectId param is missing, it gives a 400 error which is not desired.
```

Comment:
Could you file an issue for this as well?

Reply:
Sure, done.